proxies/live/apiproxy/policies/AssignMessage.OAuthPolicyErrorResponse.xml

```diff
@@ -1,23 +1,27 @@
 <AssignMessage async="false" continueOnError="false" enabled="true" name="AssignMessage.OAuthPolicyErrorResponse">
   <Set>
     <StatusCode>401</StatusCode>
-    <Payload contentType="application/json">
+    <Payload contentType="application/json" variablePrefix="%" variableSuffix="#">
       {
         "resourceType": "OperationOutcome",
+        "meta": {
+          "profile": [
+            "https://fhir.hl7.org.uk/StructureDefinition/NHSDigital-OperationOutcome"
```

Comment:
Should be https://fhir.nhs.uk/StructureDefinition/NHSDigital-OperationOutcome

As per https://nhsd-confluence.digital.nhs.uk/display/WAYF/JSON+Examples#JSONExamples-ErrorResponse and https://nhsd-confluence.digital.nhs.uk/display/WAYF/FHIR+Profiles